polyfills/Array/from/polyfill.js

```diff
@@ -1,36 +1,104 @@
-Object.defineProperty(Array, 'from', {
-	configurable: true,
-	value: function from(source) {
-		// handle non-objects
-		if (source === undefined || source === null) {
-			throw new TypeError(source + ' is not an object');
+(function arrayFromPolyfill() {
```

Comment:
A named IIFE isn't available from outside it's own scope.

This allows for the name to be available in stack traces.